addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -308,7 +308,7 @@
         <field name="name">mrp.production.work.order.calendar</field>
         <field name="model">mrp.workorder</field>
         <field name="arch" type="xml">
-            <calendar date_stop="date_planned_finished" date_start="date_planned_start" string="Operations" color="workcenter_id" event_limit="5">
```

Comment:
Fair enough to me on the principle.
Although, I would not change the true behavior by default. This code has either too much or too little.
If the attr delete is not defined, it is false, so this line is too much....

The name is simple enough, though it may interfere wit some future developments. What if we want the the delete button to appear conditionally on records ? would we have on calendar_popover another this.recordDelete param ?

I must specify that my future proof approach might at times been somewhat controversial :D

Reply:
@kebeclibre this is simple development like we support ```create=0/1 delete=0/1 edit=0/1``` if you want to display delete icon on conditional bases then it is completely different implementation as we need to evaluate that condition on that particular record if satisfy then show delete button else hide, what do you want here, would you please elaborate?

Reply:
fair enough